version.py

```diff
@@ -1 +1 @@
-__version__ = "0.5.0"
+__version__ = "0.5.1"
```

Comment:
Since you are removing `can_nlohmann_json` doesn't that signify a breaking change ?

Reply:
Sure!